### PR TITLE
Iteration6 Comments and Voting APIs incorporated

### PR DESCRIPTION
**Pull request by Shilpa-Rani**

### CODE REVIEW

File: src/server/controllers/vote.js [hunk 1]
```diff
@@ -0,0 +1,140 @@
...
+    var eventId = req.query.eventId;
+    var emailId = req.query.emailId;
+
+    Vote.find({"eventId": eventId, "emailId":emailId}).exec(function(err, singleVote) {
+        if(err) {
+            console.log("Error: unable to get events");
+            res.status(400).end();
+        } else {
+            //there will always be a single vote
+            var response = "";
```
Comment (jeffreyj100) [inline src/server/controllers/vote.js:68]: If you look at coveralls not all paths are covered. It's only hitting downvote case.

File: src/server/controllers/vote.js [hunk 2]
```diff
@@ -0,0 +1,140 @@
...
+            res.status(400).end();
+        } else {
+
+            var upVote = 0;
+            var downVote = 0;
+
+            for (var i  in listVotes){
+                // console.log(listVotes[i]);
+                if(listVotes[i].vote == 1)
+                    upVote++;
```
Comment (jeffreyj100) [inline src/server/controllers/vote.js:28]: This upvote path is not covered.

File: src/server/controllers/vote.js [hunk 3]
```diff
@@ -0,0 +1,140 @@
...
+            console.log("Error Happened");
+            return res.status(400).json({
+                errors: errors.array()
+            });
+        }
+        const {eventId, emailId, vote} = req.body;
+
+        //vote value must be +1 or -1.
+        if(vote != 1 && vote != -1)
+            return res.json({"status":"INVALID_VOTE_NUMBER"});
```
Comment (jeffreyj100) [inline src/server/controllers/vote.js:101]: This bounds check is never reached.

**Comment by Shilpa-Rani** [comment]
Iteration6 Comments and Voting APIs incorporated

**Pull request merged**